app/controllers/provider_interface/sessions_controller.rb

```diff
@@ -27,7 +27,7 @@ def sign_in_by_email
 
         SlackNotificationWorker.perform_async(
           "Provider user #{provider_user.first_name} has requested a fallback signin link",
-          edit_support_interface_provider_user_path(provider_user),
+          Rails.application.routes.url_helpers.edit_support_interface_provider_user_path(provider_user),
```

Comment:
I think the link helper works allright, but it should just be url instead of path.

```suggestion
          edit_support_interface_provider_user_url(provider_user),
```

Reply:
I found the verbose version in a different controller, so I copied it over, but you're right. The only place these don't work OOTB I think is inside component `.rb` files.